samples/cellular/nrf_cloud_multi_service/overlay_nrf7002ek_wifi_no_lte.conf

```diff
@@ -124,3 +126,6 @@ CONFIG_NET_SOCKETS_POLL_MAX=8
 # this could be set as low as 130000. Add 256 bytes for each additional scanning result, assuming
 # sane SSID lengths. We set the heap much higher in case of long SSIDs.
 CONFIG_HEAP_MEM_POOL_SIZE=130000
+
+## Configure device provision certificate
+CONFIG_NRF_CLOUD_PROVISION_CERTIFICATES=y
```

Comment:
This is not required by CMakeLists.txt -- you mention it is needed for a test, can you elaborate on that? I think this should be added to the config of that test, rather than in the nrf7002 overlay

Reply:
Okay. As we discussed over there, requiring `-DCONFIG_NRF_CLOUD_PROVISION_CERTIFICATES` to be enabled  manually was a deliberate decision, but this is not a hill I want to die on

Reply:
Sorry I'm completely not aware of the doc.

`west build --board your_board -p always -- -DCONFIG_NRF_CLOUD_PROVISION_CERTIFICATES=y -DCONFIG_NRF_CLOUD_CLIENT_ID_SRC_COMPILE_TIME=y -DCONFIG_NRF_CLOUD_CLIENT_ID="698d4c11-0ccc-4f04-89cd-6882724e3f6f"`

In my test, I only added one `-DOVERLAY_CONFIG=overlay_nrf7002ek_wifi_no_lte.conf`

Reply:
> This is not required by CMakeLists.txt -- you mention it is needed for a test, can you elaborate on that? I think this should be added to the config of that test, rather than in the nrf7002 overlay

I mean this line:
`# Include certs files if enabled.
zephyr_include_directories_ifdef(CONFIG_NRF_CLOUD_PROVISION_CERTIFICATES certs)`

As far as I know this is the only way to supply the credentials to lib_nrf_cloud for Wi-Fi devices as the TF-M PS storage is not ready yet.